provider-directory-api/src/main/java/gov/va/api/health/providerdirectory/api/resources/PractitionerRole.java

```diff
@@ -0,0 +1,223 @@
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import gov.va.api.health.providerdirectory.api.Fhir;
+import gov.va.api.health.providerdirectory.api.bundle.AbstractBundle;
+import gov.va.api.health.providerdirectory.api.bundle.AbstractEntry;
+import gov.va.api.health.providerdirectory.api.bundle.BundleLink;
+import gov.va.api.health.providerdirectory.api.datatypes.Address;
+import gov.va.api.health.providerdirectory.api.datatypes.CodeableConcept;
+import gov.va.api.health.providerdirectory.api.datatypes.ContactPoint;
+import gov.va.api.health.providerdirectory.api.datatypes.HumanName;
+import gov.va.api.health.providerdirectory.api.datatypes.Identifier;
+import gov.va.api.health.providerdirectory.api.datatypes.Period;
+import gov.va.api.health.providerdirectory.api.datatypes.Signature;
+import gov.va.api.health.providerdirectory.api.datatypes.SimpleResource;
+import gov.va.api.health.providerdirectory.api.elements.BackboneElement;
+import gov.va.api.health.providerdirectory.api.elements.Extension;
+import gov.va.api.health.providerdirectory.api.elements.Meta;
+import gov.va.api.health.providerdirectory.api.elements.Narrative;
+import gov.va.api.health.providerdirectory.api.elements.Reference;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+
+import javax.validation.Valid;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Pattern;
+import java.util.List;
+
+@Data
+@Builder
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+@AllArgsConstructor
+@JsonAutoDetect(fieldVisibility = JsonAutoDetect.Visibility.ANY)
+@Schema(
+  description = "http://www.fhir.org/guides/argonaut/pd/StructureDefinition-argo-practitionerrole.html"
+)
+public class PractitionerRole implements DomainResource {
+  @NotBlank String resourceType;
+
+  @Pattern(regexp = Fhir.ID)
+  String id;
+
+  @Valid Meta meta;
+
+  @Pattern(regexp = Fhir.URI)
+  String implicitRules;
+
+  @Pattern(regexp = Fhir.CODE)
+  String language;
+
+  @Valid Narrative text;
+  @Valid List<SimpleResource> contained;
+  @Valid List<Extension> modifierExtension;
+  @Valid List<Extension> extension;
+  @Valid List<Identifier> identifier;
+
+  Boolean active;
+
+  @Valid Period period;
+
+  @Valid @NotNull Reference practitioner;
+
+  @Valid @NotNull Reference organization;
+
+  @Valid @NotNull CodeableConcept code;
+
+  @Valid @NotNull CodeableConcept specialty;
+
+  @Valid List<Reference> location;
+
+  @Valid List<Reference> healthcareService;
+
+  @Valid List<PractitionerContactPoint> telecom;
+
+  @Valid List<Address> address;
+  @Valid Reference partOf;
+  @Valid List<OrganizationContact> contact;
```

Comment:
I don't see `address`, `partOf`, or `contact` in the spec.

Reply:
copy paste error. Fixed